src/lib/routes/admin-api/environments.ts

```diff
@@ -135,8 +156,12 @@ export class EnvironmentsController extends Controller {
             middleware: [
                 openApiService.validPath({
                     tags: ['Environments'],
+                    description: 'Toggles the environment with `name` off',
```

Comment:
Same comment here as for the `on` endpoint above: there's no summary and I'm don't understand what turning an environment on or off means.